<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>assistantSearch</class>
 <widget class="QWidget" name="assistantSearch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QToolButton" name="tbClose" >
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/buttoncancel.png</iconset>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leSearch" />
   </item>
   <item>
    <widget class="QToolButton" name="tbPrevious" >
     <property name="text" >
      <string>Previous</string>
     </property>
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/helpprev.png</iconset>
     </property>
     <property name="toolButtonStyle" >
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbNext" >
     <property name="text" >
      <string>Next</string>
     </property>
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/Icons/Icons/helpnext.png</iconset>
     </property>
     <property name="toolButtonStyle" >
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbCaseSensitive" >
     <property name="text" >
      <string>Case Sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbWholeWords" >
     <property name="text" >
      <string>Whole Words</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>tbClose</sender>
   <signal>clicked()</signal>
   <receiver>assistantSearch</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>21</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel" >
     <x>64</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
